Makefile

```diff
@@ -0,0 +1,99 @@
+include Makefile-utils.mk
+
+#
+# Calypso
+#
+.PHONY: pre-start
+pre-start: welcom versions
+
+versions: B = $(FG_BLUE)
+versions: R = $(FG_RED)
+versions: Z = $(FG_RESET)
+versions: CHECK = if [ $$WANT_$2 != $$HAVE_$2 ]; then echo -e "$Rx$Z $1 major version is $R$$HAVE_$2$Z but we need $B$$WANT_$2$Z"; exit 1; fi
+versions: $(NODE) $(NPM)
```

Comment:
I think it would be nice if this retained the old script's output of the currently running node/npm version and the ones desired by `package.json`. Generally we only want to enforce major version compatibility, but I think it's a useful debugging tool to see which version you're running versus what's required–if we're relying on a new feature somewhere in a point release or something else, it just makes things easier.

When I ran this locally I didn't see the output.

Reply:
Colorizd the major version number in b1cc9877df - now it's 17 lines